bazel/envoy_build_system.bzl

```diff
@@ -81,10 +80,8 @@ def envoy_extension_package(enabled_default = True, default_visibility = EXTENSI
         flag_values = {":enabled": "True"},
     )
 
-def envoy_mobile_package():
-    # Mobile packages should only be visible to other mobile packages, not any other
-    # parts of the Envoy codebase.
-    envoy_extension_package(default_visibility = MOBILE_PACKAGE_VISIBILITY)
+def envoy_mobile_package(default_visibility = ["//visibility:public"]):
```

Comment:
question: this looks like it'll make mobile packages visible to all of envoy, is that what we want?

Reply:
got it, makes sense then